src/index.html

```diff
@@ -12,5 +12,20 @@
   </head>
   <body>
     <h1>Search bar airbnb</h1>
+  <form action="" method="POST">
+    <input
+      class="search"
+      name="search-big"
+      data-qa="big"
+      data-qa="keypress"
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot;">
+
+      <input
+      class="search small"
```

Comment:
```suggestion
      class="search search--small"
```
тогда в search будут все общие стили, а в search--small - измененный размер